app/components/mappingsTable/mappingsTable.component.js

```diff
@@ -0,0 +1,224 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+mappingsTable.$inject = ['openmrsRest', 'openmrsNotification'];
+
+export default function mappingsTable(openmrsRest, openmrsNotification){
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This function has too many statements. (43)](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/file/2741260425/issues/source?bid=3196969&fileBranchId=3354756#l12)